api/TestApi/TestJsonServer.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.IO;
+using System.Net;
+using System.Net.Sockets;
+using System.Threading;
+using NUnit.Framework;
+using SynthesisAPI.AssetManager;
+using SynthesisAPI.Utilities;
+
+namespace TestApi {
+    [TestFixture]
+    public static class TestJsonServer {
+        [Test]
+        public static void TestReceivingData() {
```

Comment:
Please use this unit test function I've set up for you to test your server and make sure it can receive and deserialize your json data.